tools/pythonpkg/src/numpy/numpy_scan.cpp

```diff
@@ -411,92 +388,4 @@ void VectorConversion::NumpyToDuckDB(PandasColumnBindData &bind_data, py::array
 	}
 }
 
-void VectorConversion::BindPandas(const DBConfig &config, py::handle df, vector<PandasColumnBindData> &bind_columns,
```

Comment:
Sadly git cant recognize this is moved to `src/pandas/bind.cpp`
Going to cause some merge conflicts, so this PR should not be merged before the current PRs that deal with this are merged (such as #6523 )